test/tests/functional/pbs_sched_runjobwait.py

```diff
@@ -106,7 +106,9 @@ def test_jobrunwait_throughput_clash(self):
         # Setting job_run_wait to 'none' should just delete TP
         self.server.manager(MGR_CMD_SET, SCHED,
                             {'job_run_wait': "none"}, id="default")
-        self.server.expect(SCHED, 'throughput_mode', op=UNSET, id="default")
+        rt = self.server.status(SCHED)
+        self.assertNotIn('throughput_mode', rt[0].keys(),
```

Comment:
Just for my understanding, was expect() for UNSET not working?

Reply:
By calling expect() with UNSET, the framework expected the default value which is "True"; whereas the case to be checked was absence of attribute "throughput_mode"; hence it was failing.